gcc/config/riscv/bitmanip.md

```diff
@@ -153,6 +153,14 @@
   "sbsetw\t%0,%1,%2"
   [(set_attr "type" "bitmanip")])
 
+(define_insn "*sbsetiw"
+  [(set (match_operand:DI 0 "register_operand" "=r")
+	(ior:DI (sign_extend:DI (match_operand:SI 1 "register_operand" "=r"))
```

Comment:
Redundant `=` for input operand, which is reported by Evgeny before: https://github.com/riscv/riscv-bitmanip/issues/79